realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -198,77 +200,77 @@ public static void validateTable(ImplicitTransaction transaction) {
         if(transaction.hasTable("class_AllTypes")) {
             Table table = transaction.getTable("class_AllTypes");
             if(table.getColumnCount() != 9) {
```

Comment:
Now that the generator is updated anyway, can you please make spacing consistent and add " " between "if" and "(" and "for" and "("?